src/layer/const.ts

```diff
@@ -1,8 +1,8 @@
 import { Polygon, MultiPolygon } from '@turf/helpers';
 
-import { BBox } from 'src/bbox';
-import { CRS_IDS } from 'src/crs';
-import { Effects } from 'src/mapDataManipulation/const';
+import { BBox } from '../bbox';
+import { CRS_IDS } from '../crs';
+import { Effects } from '../mapDataManipulation/const';
```

Comment:
Since `BBox` and `Effects` are exported in `src/index.ts`, could they be imported from there?